libsweep/include/protocol.h

```diff
@@ -6,7 +6,7 @@
  * Implementation detail; not exported.
  */
 
-#include <stdint.h>
+#include <cstdint>
```

Comment:
I never fully understood the difference here - I thought if we want to use `int32_t` (unprefixed) we have to include `stdint.h`, and if we include `cstdint` we _have to_ use `std::int32_t` to be on the safe side?